packages/graphql/src/utils/execute.ts

```diff
@@ -79,31 +79,29 @@ const getExecutor = (input: {
     defaultAccessMode: SessionMode;
     context: Context;
 }): { driver?: Driver; session?: Session; transaction: Transaction } => {
-    if (!input.context.executionContext) {
+    const executionContext = input.context.executionContext;
+    
+    if (!executionContext) {
         const driver = input.context.driver;
         const session = driver.session(getSessionParams(input));
         const transaction = session.beginTransaction(getTransactionConfig());
         return { driver, session, transaction };
     }
 
-    const executionContextType = Object.getPrototypeOf(input.context.executionContext).constructor.name;
 
-    if (executionContextType === "Driver") {
-        const driver = input.context.executionContext as Driver;
-        const session = driver.session(getSessionParams(input));
+    if (executionContext instanceof Driver) {
+        const session = executionContext.session(getSessionParams(input));
```

Comment:
TS2339: Property 'session' does not exist on type 'Driver | Session | Transaction'.   Property 'session' does not exist on type 'Session'.